kafka-connect-ftp/build.gradle

```diff
@@ -32,8 +32,11 @@ project(":kafka-connect-ftp") {
     compile project(':kafka-connect-common')
     compile "commons-net:commons-net:${commonsnetVersion}"
     compile "commons-codec:commons-codec:${commonsCodecVersion}"
+    compile "com.jcraft:jsch:0.1.55"
```

Comment:
how confident are we on the maintainability of whatever libraries were used for this (last update on 2018)